code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -250,7 +250,7 @@
 	name = "Ketrazine"
 	id = "ketrazine"
 	results = list("ketrazine" = 3)
-	required_reagents = list("epinephrine" = 1, "ephedrine" = 1, "tricordazine" = 1, "atropine" = 1)
+	required_reagents = list("epinephrine" = 1, "ephedrine" = 1, "tricordrazine" = 1, "atropine" = 1)
```

Comment:
A case of not testing code I see

Reply:
he keeps getting told too aaaaaaaaa

Reply:
Okay, one, I did fuck up, but in my defense, it was because I was an idiot earlier and when I was messing with tricord to get a sense of how it worked, I accidentally changed its id! So I was stupid, but I did test it! I just didn't test my testing. Which is worse. Mea culpa.